index.js

```diff
@@ -1,4 +1,101 @@
-//  in this file create an express application - use the middle-ware built into express
-//  to serve up static files from the public directory (index.html and client.js - you 
-//  can also serve up css files from the public directory this way if you want)
-//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks/:name'
\ No newline at end of file
+/*
+	Scott Henderson
+	JSCRIPT 300 A Spring 2016
+*/
+
+// in this file create an express application - use the middle-ware built into express
+// to serve up static files from the public directory (index.html and client.js - you
+// can also serve up css files from the public directory this way if you want)
+// you need to support a "/trucks" endpoint, and a dynamic route for "/trucks/:name"
+
+var express = require("express");
+var _ = require("lodash");
```

Comment:
lodash is missing from the dependencies section of your package.json file, meaning the server crashes when trying to run it because it can't find the module